docs/data-point-definition.md

```diff
@@ -8,6 +8,8 @@ Gro defines a "data point" as a discrete result produced by our API. When using
   'value': 0.131714797651,
   'unit_id': 851,
   'reporting_date': None,
+  'metadata': {},
+  'available_date': None,

```

Comment:
I see, we are pushing both the documentation updates and stage examples in the same PR

Reply:
I changed this branch to drop the documentation updates now.